product_docs/docs/biganimal/release/using_cluster/05c_upgrading_log_rep.mdx

```diff
@@ -0,0 +1,223 @@
+
+!!! Important
+Depending on where your older and newer versioned BigAnimal instances are located, this procedure may accrue ingress and egress costs from your cloud service provider (CSP) for the migrated data. Please consult your CSP's pricing documentation to see how ingress and egress fees are calculated to determine any extra costs.
+!!!
+
+### Create a new BigAnimal instance
+
+To perform a major version upgrade, create a BigAnimal instance with your desired version of Postgres. This is your target instance.
+
+Ensure your target instance is provisioned with a storage size equal to or greater than your source instance.
+
+Considering these caveats, [create your target BigAnimal instance](../getting_started/creating_a_cluster.mdx).
+
+### Gather instance information
+
+Use the BigAnimal console to obtain the following information for your source and target instance:
+
+- Read/write URI
+- Database name
+- Username 
+- Read/write host
+
+Using the BigAnimal console:
+
+    1. Select the **Clusters** tab.
+    1. Select your source instance.
+    1. From the Connect tab, obtain the information from **Connection Info**.
+
+Repeat the steps for your target instance.
+
+### Confirm the Postgres versions before migration
+
+Confirm the Postgres version on both your source and target BigAnimal instances:
+
+```
+psql "<biganimal_read/write_uri>" -c "select version();"
+```
+
+Output using Postgres 16:
+
+```
+                                                               version                                                               
+-------------------------------------------------------------------------------------------------------------------------------------
+ PostgreSQL 16.2 (Debian 16.2.0-3.buster) (BigAnimal Edition) on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
+(1 row)
+```
+
+### Migrate the database schema 
+
+On your source instance, view the details of the schema to be migrated:
+
+```sql
+/dt+;
+```
+
+Here is a sample database schema for this example:
+
+```
+                                           List of relations
+ Schema |       Name       | Type  |   Owner   | Persistence | Access method |    Size    | Description 
+--------+------------------+-------+-----------+-------------+---------------+------------+-------------
+ public | pgbench_accounts | table | edb_admin | permanent   | heap          | 1572 MB    | 
+ public | pgbench_branches | table | edb_admin | permanent   | heap          | 8192 bytes | 
+ public | pgbench_history  | table | edb_admin | permanent   | heap          | 0 bytes    | 
+ public | pgbench_tellers  | table | edb_admin | permanent   | heap          | 120 kB     | 
+```
+
+Use pg_dump with the `--schema-only` flag to copy the schema from your source to your target instance. For more information on using `pg_dump`, [see the Postgres documentation](https://www.postgresql.org/docs/current/app-pgdump.html).
```

Comment:
I think pg_dump needs <code> tag everywhere.

Reply:
Not sure what you mean. I have tested this procedure and it seems to work correctly. Can you say more? The subject-matter experts I worked with on this didn't mention anything about it.

Reply:
this can be disregarded.  No code tag needed for pg_dump.